README.md

```
@@ -105,128 +94,63 @@ Unlock the power of semantic embeddings with Qdrant, transcending keyword-based
-            Anomaly Detection
-        </td>
-    </tr>
-</table>
-
-</details>
-
 ## API
 
 ### REST
 
 Online OpenAPI 3.0 documentation is available [here](https://api.qdrant.tech/).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)